### PR TITLE
Extra toppings

### PR DESCRIPTION
**Pull request by zengofran**
Let us think of more toppings?

### CODE REVIEW

**Comment by zengofran** [comment]
Can we add Pepperoni?

File: ultimate_pizza [hunk 1]
```diff
@@ -16,7 +16,6 @@
 ###**Toppings**
 - Tomatoes
 - Cheese
--
```
Comment (thacilima) [inline ultimate_pizza]: `Mushrooms`

File: ultimate_pizza [hunk 2]
```diff
@@ -16,7 +16,6 @@
 ###**Toppings**
 - Tomatoes
 - Cheese
```
Comment (EoinFarrell) [inline ultimate_pizza:18]:
```suggestion
- Cheese
- Pepperoni
```

Comment (thacilima) [inline ultimate_pizza:18]:
```suggestion
- Cheese
- Mushrooms
```

**Pull request merged**